src/platform/forms/save-in-progress/reducers.js

```diff
@@ -132,6 +132,7 @@ export function createSaveInProgressInitialState(formConfig, initialState) {
     prefillTransformer: formConfig.prefillTransformer,
     trackingPrefix: formConfig.trackingPrefix,
     additionalRoutes: formConfig.additionalRoutes,
+    formErrors: formConfig.formErrors,
```

Comment:
The `checkValidation` function sets it:

```js
this.props.setFormErrors({
  rawErrors: errors,
  errors: reduceErrors(errors, pageList),
});
```

Including `rawErrors` may not be necessary since the list is so big.

The `errors` value is processed in the `/utilities/data/reduceErrors` function to standardize the different error messages, which will later be used for display.

Reply:
Oh wait... that's in the reducer... Huh. I have no idea. LOL